etna/ciim/constants.py

```diff
@@ -53,12 +53,13 @@ class Bucket:
     is_current: bool = False
     results: List[Any] = None
 
+    # Fetch more 'groups' so that we receive counts for any bucket/tab options we might be showing Ex: ["group:30"]
```

Comment:
Sorry @TNA-Allan - this comment does not make sense here. When I suggested updating the dataclass, I meant adding something like: 

# By default, 10 items of each aggregation are requested from the API. This can be overridden by using a string in the format '{name}:{number_of_items}'

Reply:
Why don't you want to say about `group:30`, as it seems quite out of the blue using that number without any reason. Thoughts?

Reply:
I can tell you're getting frustrated. Can I suggest taking your lunch and looking at this with fresh eyes when you come back. This value-specific comment simply makes no sense on a dataclass.

Reply:
He He .. I'm good @ababic, Its confusing when at one instant we use `:30` and say why, _then when putting the same comment when its called - ( explanation is not needed there)_ we loosing such an important comment which was previously there to know why its `:30` and not any other number. It's a random number - yes, but this number gets the desired buckets counts we are displaying. Is there a way not to loose this comment?

Reply:
From my perspective, we aren't losing anything important. The dataclass explains what is expected of instances, and the instances obey the restrictions dataclasses API. If you wanted to understand why the ":30" was there previously, there was nothing to make that clear. But now, you would just look at the dataclass, see the comment, and understand.

Reply:
My thoughts are, while refactoring my be fine, removing an existing commented code which does not belong to the code change for this PR does not feel right, besides someone would know why that number is :30  and not :10 or :15.

Reply:
Updated for default aggregations and comments [e174551c](https://github.com/nationalarchives/ds-wagtail/pull/808/commits/e174551cb7db57d32fa5e682625de2911f931675)